poi.config.js

```diff
@@ -19,7 +19,8 @@ const devServer = {
   },
   // Override using: `npm run dev:client -- --port <number>`
   port: 8081,
-  hotEntries: ['admin', 'student']
+  hotEntries: ['admin', 'student'],
+  historyApiFallback: config.histroyApiFallbackOptions
```

Comment:
Question, docs say this should be a boolean (https://poi.js.org/config.html#devserver-historyapifallback) so it might be better to send useHistoryApiFallback here?

Reply:
As usual documentation sucks but relevant source code snippets reveals (ugly) truth: https://github.com/egoist/poi/blob/38ff0e3f17513386d33fbe60a9f0d845558d0417/core/poi/lib/utils/validateConfig.js#L97

Reply:
@marinko-peso As @vladimyr  stated, `historyApiFallback` is the same thing as [webpacks](https://webpack.js.org/configuration/dev-server/#devserverhistoryapifallback) `historyApiFallback`. Docs are not clear. I'll make a PR to `poi`.

Reply:
New docs are not available for contribution.  I raised an issue on the old repository. 
https://github.com/egoist/poi.js.org/issues/8

Reply:
PR made: https://github.com/poi-bundler/website/pull/15